### ISSUE TITLE
Revert #1806 to fix QandA formatting

### ISSUE THREAD

**Issue by flannery-denny** [bootstrapworld/curriculum#1835]
Where did this formatting in Measures of Center come from?! Teachers should never see `Q` or `A`!
<img width="838" alt="Screen Shot 2024-02-08 at 12 01 53 PM" src="https://github.com/bootstrapworld/curriculum/assets/66691020/e427237e-18a6-448c-b279-80089f621ddf">

We want the OG spec:
- Q should render as a solid bullet.
- A should render as a hollow indented bullet with italicized text.

**Comment by schanzer** [comment]
@flannery-denny this is the CSS I added to handle your other request for nested bullets. Are you saying you want us to remove everything we did to make those work?

**Comment by schanzer** [comment]
I'm also unclear on why you want to show teachers questions and answers, but not label which is which for clarity

**Comment by ds26gte** [comment]
We may need to increase the list indentation for `<li>` inside a `.q-and-a`, at least for the first sublist, so that it can never be confused for an answer. Then set the block indentation for `.q-and-a-a` to a value of about half of this and include an explicit custom-made bullet that signals "Answer". (Remember, answers are no longer subitems of questions in this new régime.)

**Comment by schanzer** [comment]
@ds26gte I think Flannery has changed her mind about #1806 , and no longer needs any of this. But I want to make sure we hear from her first, before we toss out all that code.

**Comment by ds26gte** [comment]
Note that in the old régime, we can still have subitems in answers, just not in questions. If that's an acceptable situation, we can revert.

**Comment by flannery-denny** [comment]
1. The request was to add third level bullets. Instead everything that used to work exactly as specced through a series of meetings was removed and replaced... and the thing I requested wasn't working. Three level bullets looks like this in almost every version of bulleting I've ever seen:
<img width="220" alt="Screen Shot 2024-02-09 at 10 03 12 AM" src="https://github.com/bootstrapworld/curriculum/assets/66691020/eec44527-1d45-4b0d-b6ca-e35db6248fe4">
2. If third level bullets worked, that would be great. But third level bullets don't currently work (notice how the subbullets were rendering as a single runon line rather than subbulets in this image I posted on #1714)
3. QandA shouldn't be labeled because:
- the request was to add a third level of bullets, not change the first and level bullets
- it's a ton of visual clutter that makes it hard to read the content
- they aren't all actually Qs or As.
4. The current formatting is so far removed from what I should be seeing that I can't actually complete the job I am working on.

I would love to see third level bullets working, but it appears to be more complicated than I had imagined it to be. I can live without them if it's easier to revert. If this comment somehow makes the request clearer and it's easy to get from where we are to things working as expected, even better.

But leaving QandA rendering with Qs and As and no indentation is getting in the way of me being able to do my work.

**Comment by flannery-denny** [comment]
@schanzer
1. The request was to add third level bullets. Instead everything that used to work exactly as specced through a series of meetings was removed and replaced... and the thing I requested wasn't working. Three level bullets looks like this in almost every version of bulleting I've ever seen:
<img width="220" alt="Screen Shot 2024-02-09 at 10 03 12 AM" src="https://github.com/bootstrapworld/curriculum/assets/66691020/eec44527-1d45-4b0d-b6ca-e35db6248fe4">

2. If third level bullets worked, that would be great. But third level bullets don't currently work (notice how the subbullets were rendering as a single runon line without any bullets rather than subbulets in this image I posted on #1714)

![image](https://github.com/bootstrapworld/curriculum/assets/66691020/bb5bcec1-0810-475e-a5b6-950ff7811a5c)


4. QandA shouldn't be labeled because:
- the request was to add a third level of bullets, not change the first and second level bullets
- it's a ton of visual clutter that makes it hard to read the content
- they aren't all actually Qs or As.
5. The current formatting is so far removed from what I should be seeing that I can't actually complete the job I am working on.

I would love to see third level bullets working, but it appears to be more complicated than I had imagined it to be. I can live without them if it's easier to revert. If this comment somehow makes the request clearer and it's easy to get from where we are to things working as expected, even better.

But leaving QandA rendering with Qs and As and no indentation is getting in the way of me being able to do my work.
